components/Hyauctions/Hyauctions.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function Hyauctions() {
+    return (
+        <>
+            <h2>Coflnet</h2>
+            <hr />
+            <p>Seems you were looking for Hyauctions. We also provide a auction history to check prices and player auctions.</p>
+            <p>Just try using the searchbar above.</p>
```

Comment:
```suggestion
            <p>Just try using the search bar above. (it actually works)</p>
```
poke fun at the broken search of hyauctions